tests/sitemaps_tests/test_http.py

```diff
@@ -52,6 +71,18 @@ def test_simple_sitemap_section(self):
 """ % (self.base_url, date.today())
         self.assertXMLEqual(response.content.decode(), expected_content)
 
+    def test_simple_sitemap_no_section(self):
+        response = self.client.get('/simple/sitemap-simple2.xml')
+        self.assertEqual(response.status_code, 404)
```

Comment:
You can check `str(response.context['exception'] is '"No sitemap available for section..."` to improve these 404 tests.